package.json

```diff
@@ -1,19 +1,15 @@
 {
   "private": true,
   "scripts": {
+    "prepare": "husky install",
```

Comment:
@vapurrmaid FYI, I'm going to update this to `husky` to match the monorepo :smile: